examples/window/window_settings.rs

```diff
@@ -10,14 +10,14 @@ use bevy::{
 fn main() {
```

Comment:
Found a bug: If I click and drag a window and continue to hold my click, the window starts drifting toward the upper right hand corner.

Reply:
I can't seem to repro this on my laptop (macos)/desktop (windows), not entirely sure if I'll be able to fix this issue myself

Reply:
I'm going to take a guess at this and say maybe it is drifting due to some rounding weirdness of setting cursor position.